emsdk.bat

```diff
@@ -1,5 +1,3 @@
-@SET PREVPATH=%PATH%

```

Comment:
Hmm, I know I had some grand reason to guarding the `PATH` variable before and after, but searching the commit history, I am unable to find it. This may have been a leftover, perhaps I had at some point appended the directory to PYTHON to PATH, instead of invoking it explicitly.